internals/secrethub/run.go

```diff
@@ -667,13 +676,6 @@ type EnvDir map[string]string
 // NewEnvDir sources environment variables from files in a given directory,
 // using the file name as key and contents as value.
 func NewEnvDir(path string) (EnvDir, error) {
-	_, err := os.Stat(path)
-	if os.IsNotExist(err) {
-		return nil, ErrEnvDirNotFound
-	} else if err != nil {
-		return nil, ErrReadEnvDir(err)
-	}
-
```

Comment:
Removed these checks because `NewEnvDir` is called like this:
```go
_, err = cmd.osStat(envDir)
if err == nil {
	dirSource, err := NewEnvDir(envDir)
	if err != nil {
		return nil, nil, err
	}
	envSources = append(envSources, dirSource)
}
```

So the error will always be nil.